ivadomed/loader/utils.py

```diff
@@ -254,7 +255,7 @@ def imed_collate(batch):
         if elem.shape == ():  # scalars
             py_type = float if elem.dtype.name.startswith('float') else int
             return __numpy_type_map[elem.dtype.name](list(map(py_type, batch)))
-    elif isinstance(batch[0], int_classes):
+    elif isinstance(batch[0], int): #int_classes):
```

Comment:
does this change have to do with the purpose of this PR? if so, how? if not, can you please fix it in another PR?

Reply:
Some versions of `torch` do not have the `int_classes` in the `torch._six` attribute, and according to  this [discussion](https://discuss.pytorch.org/t/cannot-import-name-int-classes-from-torch-six/126611) users can fix that by simply using `int` instead of importing `int_classes`

Reply:
looks a bit out-of-scope-- i will let @dyt811 take over

Reply:
Okay. We may add it in a future PR since it works for all torch versions, unlike the `int_classes` .

Reply:
in this case can you pls open an issue so it doesn't fall through the cracks-- thx

Reply:
Yes, please.